wandering-warriors/main.py

```diff
@@ -0,0 +1,26 @@
+from kivy.app import App
+from kivy.uix.widget import Widget
+from kivy.lang import Builder
+
+
+class Abacus(Widget):
+    pass
+
+
+class DrawingInput(Widget):
```

Comment:
I'm not going to be picky about naming conventions once we get deeper into the code, but I think it's worth putting "cuneiform" and/or "ledger" in the name here as "drawing input" doesn't really give off the right connotation. My recommendations would be any of "CuneiformDrawingInput", "CuneiformInput", "LedgerDrawingInput", "LedgerInput"

Reply:
My reasoning is that at a functional level, the space for taking drawing input is general, could be multi purpose.  It's taking user drawn input, converting it to an analyzable data structure, and passing it off.  Could be cuneiform, or something else.  Thats for the classifier to decide.

Reply:
We're not working in a general sense though. This is for the context of this project. This is just one of the three main widgets, and it is the cuneiform/ledger input